### DISCUSSION TITLE
vcpkg

### DISCUSSION THREAD

**Discussion by machalejm** [swig/swig#3044] [General]
HI! Possibly a stupid question, but is it feasible to get SWIG published onto vcpkg?

**Comment by wsfulton** [comment]
I thought vcpkg was for libraries. Maybe SWIG is needed to create those libraries, in which case there might be a case to package SWIG up into vcpkg. This is a question best answered by the vcpkg project.

There are dozens of packaging projects and the SWIG developers do not as a rule get involved in packaging up binary distributions. That is left to the upstream package projects. We publish source tarballs which package managers build in the manner that they want. Windows binaries is the only exception given the history of Windows and ad-hoc manner in which Windows users have to obtain software, so SWIG also provides a Windows zip file with the swig executable in it.
